Mention in README.md about run-time error due to missing libs

oleg-alexandrov opened the pull request:
I installed the s2p dependencies with conda as I don't have root access to my system and pip cannot install libfftw3 on its own.  After compiling s2p with pip, it could not find the libraries. 

Then s2p throws error 127 which can be a bit mystifying. I added a note saying that one should set for the current call LD_LIBRARY_PATH. 

I know this is not an ideal solution, but I don't know any better way.

jmichel-otb commented on s2p.py:
```diff
@@ -194,8 +194,10 @@ def stereo_matching(tile,i):
+
+    # verifying non-epipolar_rectification is possible
+    if cfg['matching_algorithm'] not in ['tvl1']:
+        cfg['epipolar_rectification'] = True
```
I would prefer to avoid modifying cfg parameters on the fly (without warning the user). We can check in initalization for compatibility between
 cfg['matching_algorithm'] and cfg['epipolar_rectification'] and output a nice error message instead.

jmichel-otb commented on s2plib/config.py:
```diff
@@ -55,6 +55,9 @@
 cfg['subsampling_factor'] = 1
 
+# switch to False to disable epipolar rectification and rectify the secondary tiles only
+cfg['epipolar_rectification'] = True
```
Since not all combination of cfg['epipolar_rectification'] and cfg['matching_algorithm'] are possible, maybe we can remove the cfg['epipolar_rectification'] parameter and derive it from cfg['matching_algorithm'] instead ? For algorithms that can be used for both 1D and 2D, like tvl1 we can have two cfg['matching_algorithm'], such as tvl1_epi and tvl1_2d ?

oleg-alexandrov committed:
Update README.md

I installed the s2p dependencies with conda as I don't have root access to my system and pip cannot install libfftw3 on its own.  After compiling s2p with pip, it could not find the libraries. 

Then s2p throws error 127 which can be a bit mystifying. I added a note saying that one should set for the current call LD_LIBRARY_PATH. 

I know this is not an ideal solution, but I don't know any better way.

mnhrdt committed:
Update README.md
